app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -831,7 +831,7 @@ private void processCalibrationNoUI(final double glucosenumber, final double tim
         }
     }
 
-    static void startIntentThreadWithDelayedRefresh(final Intent intent) {
+    public static void startIntentThreadWithDelayedRefresh(final Intent intent) {
```

Comment:
I'm not seeing this used in your code? Is that correct? If so please revert the visibility of this method. Thanks

Reply:
That's reverted.